packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -394,4 +565,56 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
   );
 };
 
+type MultipleComboboxProps<T> = {
+  selectedItems?: SelectOption<T>[];
+  selectedItemsRenderer?: (options: SelectOption<T>[]) => JSX.Element;
+  tagRenderer: (item: SelectOption<T>) => JSX.Element;
+  mRef: React.ForwardedRef<HTMLDivElement>;
+};
+
+const MultipleSelectedContainer = <T,>({
```

Comment:
this should go in a separate file. I think that would also help clean up the code in main ComboBox component, and more things would be handled through Context